doc/tasks/twigcs.md

```diff
@@ -0,0 +1,48 @@
+#  TwigCs
+
+Check Twig coding standard based on [Allocine/TwigCs](https://github.com/allocine/twigcs) .
+
+***Composer***
+
+```
+composer require --dev allocine/twigcs
+```
+
+***Config***
+
+The task lives under the `twigcs` namespace and has following configurable parameters:
+
+```yaml
+# grumphp.yml
+parameters:
+    tasks:
+        twigcs:
+            path: '.'
+            severity: 'warning'
+            ruleset: 'Allocine\Twigcs\Ruleset\Official'
+            triggered_by: ['twig']
+```
+
+**path**
```

Comment:
@veewee, rfc:
```
**path**

*Default: null*

By default `.` wil be used.
You can specify an alternate location by changing this option.
```

Reply:
Hello @Landerstraeten,

Thank you for this feedback, i changed it :smile: